README.md

```diff
@@ -131,3 +132,10 @@ virsh console domain_name
 ```
 
 To get to the bootstrap node. The username is `core` and the password is `notworking`
+
+### Determining if your filesystem supports d_type
+At the command line, try:
+```
+xfs_info /mount-point
```

Comment:
does it work for other types of FS?

Reply:
I think this is the primary OS where this is a problem since default CentOS didn't enable until 7.4

Reply:
Perhaps I am just clueless, but from the documentation we have it's not clear to me if I should check / call `xfs_info` on any other file system than XFS created in pre CentOS-7.5. Maybe it's well known that d_type thingy applies to XFS only, then I am sorry for the noise. But I have no idea what it is and how to make sure this step in troubleshooting doesn't apply to me if I have a different FS.